### PR TITLE
refactor(angular): textarea merge

### PR DESCRIPTION
**Pull request by arciela**

### CODE REVIEW

File: src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.sbb-textarea {
+  min-height: 6rem !important;
```
Comment (kyubisation) [inline src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.css:2]: Is the important now really necessary?

**Review by kyubisation — CHANGES_REQUESTED** [review]
A few minor issues. Other than that it looks near completion. 👍

File: src/angular/textarea/public-api.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export * from './textarea.module';
+export * from './textarea/textarea.component';
+/** @deprecated Remove with v12 */
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/angular/textarea/public-api.ts]: done :)

File: src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.css (hunk 1 above)
Comment (jeripeierSBB) [reply src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.css:2]: seems do depend whether we test no @arciela s computer or mine. Will investigate further

Comment (jeripeierSBB) [reply src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.css]: not needed anymore

**Review by jeripeierSBB — APPROVED** [review]
LGTM :)

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Thank you for your work!

**Pull request merged**